src/ispc.cpp

```diff
@@ -1346,6 +1356,8 @@ Target::Target(Arch arch, const char *cpu, ISPCTarget ispc_target, bool pic, boo
 #else
                 CPUID = CPU_AppleA13;
 #endif
+            } else if (g->target_os == TargetOS::windows) {
+                CPUID = CPU_MicrosoftSQ1
```

Comment:
```suggestion
                CPUID = CPU_MicrosoftSQ1;
```

Reply:
I get the following warning with this:
```
'microsoft-sq1' is not a recognized processor for this target (ignoring processor)
```
I think it should be fine to just remove this if block and have CPUID be CPU_CortexA35 like suggested above.